README.md

```diff
@@ -1,6 +1,7 @@
 # Deis Client
 
 [![Build Status](https://travis-ci.org/deis/workflow-cli.svg?branch=master)](https://travis-ci.org/deis/workflow-cli)
+[![Build status](https://ci.appveyor.com/api/projects/status/gduy4urj0q6lnrxx?svg=true)](https://ci.appveyor.com/project/mboersma/workflow-cli-fgqd7)
```

Comment:
not really sure how appveyor works - is there a way to put this under a `deis` project, instead of `mboersma`?